README.md

```diff
@@ -1,2 +1,18 @@
 # tilelive-AsMVT
+
 Tilelive module for generating Mapbox Vector Tiles from a Postgres DB using ST_AsMVT.
+
+Requires a postgres function that takes 3 int arguments(zoom, x, y) and returns vector tile data to exist.
```

Comment:
It would be nice to include an example of what a function that leverages `ST_AsMVT` might look like